service/worker/indexer/processor.go

```diff
@@ -232,7 +232,7 @@ func (p *indexProcessor) addMessageToES(indexMsg *indexer.Message, kafkaMsg mess
 }
 
 func (p *indexProcessor) generateESDoc(msg *indexer.Message, keyToKafkaMsg string) map[string]interface{} {
-	doc := p.dumpFieldsToMap(msg.Fields)
+	doc := p.dumpFieldsToMap(msg.Fields, msg.GetDomainID())
```

Comment:
mind just ensuring this is nil safe? Might be worth a small test

Reply:
nil condition is handled in the GetDomainID() function. It will return default empty string it domain id is nil